demo/image_demo.py

```diff
@@ -1,26 +1,31 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+import os
 from argparse import ArgumentParser
 
 import mmcv
 from mmdet.apis import inference_detector, init_detector
+from mmengine.utils import ProgressBar, scandir
 
 from mmyolo.registry import VISUALIZERS
 from mmyolo.utils import register_all_modules
 
 
 def parse_args():
     parser = ArgumentParser()
-    parser.add_argument('img', help='Image file')
+    parser.add_argument(
+        'source', help='Image source, include image file, dir and URL.')
     parser.add_argument('config', help='Config file')
     parser.add_argument('checkpoint', help='Checkpoint file')
-    parser.add_argument('--out-file', default=None, help='Path to output file')
+    parser.add_argument(
+        '--out-path', default='./output', help='Path to output file')
+    parser.add_argument(
```

Comment:
Is it necessary to keep this parameter?

Reply:
I think we need to, sometime we need to compare different model then we need to set different out put dir